README.md

```diff
@@ -1,5 +1,84 @@
-# Overview
+# COMPSCI 377 LAB #1: Inverted Index
+Professor Tim Richards
 
-# Instructions
+TAs Matthew Gregory, Priyanka Mary Mammen, and Bin Wang
 
-# Go see Gradescope
+## Purpose
+This lab is designed to make you more familiar with C++, the language you will need to program with in this course., and to ensure you are familiar with the Edlab environment. You are expected to already be familiar with essential programming techniques taught in previous classes like CS 187 and CS 220. Each lab in this course is counted as part of your lab grade in this class, which totals to 10% of your final grade.
+
+Please submit your answers to this lab on Gradescope in the assignment marked “Lab #1’. All answers are due by the end of discussion, approximately 50 minutes after it has started. The TA present in your lab will do a brief explanation of the various parts of this lab, but you are expected to answer all questions by yourself. Please raise your hand if you have any questions during the lab section. Questions and Parts have a number of points marked next to them to signify their weight in this lab’s final grade.
+
+## Part 1: Edlab (10 Points)
+First, you will need to show that you know how to successfully access the Edlab environment that we will be using for our programming projects this semester. Using a secure shell such as this app for Google Chrome should suffice for the access. Connect to _____@elnux.cs.umass.edu, where the blank space is replaced by your UMass student username (the same one used for SPIRE and Moodle). Click on the ‘SFTP’ option to upload files. After it connects, it should ask you for your password, which you input to log in. If you do not already have a password, please create a new one – the default is set to Edaaabbb, where ‘aaa’ is the last 3 digits of your student ID and ‘bbb’ is the first three letters of your student username.
```

Comment:
Looks like you missed the link to the secure shell extension. I think
```
such as this app for Google Chrome
```
should be 
```
such as [this extension](https://chrome.google.com/webstore/detail/secure-shell/iodihamcpbpeioajjeobimgagajmlibd) for Google Chrome
```
I choose the word extension because [Chrome Apps have be deprecated](https://developer.chrome.com/docs/apps/migration/)